### PR TITLE
Lei civic api client

### PR DESCRIPTION
**Pull request by LeiChen0218**
Add drug check function and print out error items with given evidence type.

Here's the usage for drug check:
Specify evidence types and print on web page(Predictive,for example):
civic-api-client evidence-items-list --drug --evi_type Predictive —web

### CODE REVIEW

File: Drug_check_usage.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Here's the usage for drug search:
```
Comment (gatoravi) [inline Drug_check_usage.txt:1]: Hi @LeiChen0218 , do we need this file? The usage could perhaps go in the README?

File: civic_api_client/evidence_items_lister.py [hunk 2]
```diff
@@ -67,18 +81,62 @@ def check_doid(self, variant_id, variant_detail, evidence_items):
...
-            self.display_invalid_web()
+            self.display_invalid_eis_web()
         else:
             sys.stderr.write("Printing invalid DOIDs\n")
             print "\nDOID\tvariant_ID\tvariant_civic_url"
             for ei1 in self.invalid_eis:
-                print str(ei1.doid) + "\t" + str(ei1.variant_id) + "\t" + ei1.variant_civic_url + "\n"
+                if ei1.error == "DOID":
+                    print str(ei1.evi_id) + "\t" + str(ei1.doid) + "\t" + str(ei1.variant_id) + "\t" + ei1.variant_civic_url + "\n"
+            sys.stderr.write("Pringting invalid drug names\n")
```
Comment (gatoravi) [inline civic_api_client/evidence_items_lister.py]: "Pringting" instead of "Printing" in a couple of places :tophat:

**Comment by malachig** [comment]
@LeiChen0218 great!

As discussed, in the future you can name you branch to refer to its purposes (e.g. feature_x or bug_y). Everything you do in git is linked to your account, so no need to indicate this in the branch name.  Once merged, the branch can be deleted.

**Comment by malachig** [comment]
@LeiChen0218 some suggestions for this function before merging:
Example query `civic-api-client evidence-items-list --drug --max-gene-count 10`

1.) Do not end each entry with an extra newline character.  Just display one result per line
2.) The URL should resolve to the evidence item, not just the variant
3.) For consistency option `--evi_type` should be `--evi-type`
4.) Display the gene and variant name in each row of output

**Comment by LeiChen0218** [comment]
Yes, sir.

On Feb 18, 2016, at 1:05 PM, Malachi Griffith <notifications@github.com<mailto:notifications@github.com>> wrote:

@LeiChen0218https://github.com/LeiChen0218 great!

As discussed, in the future you can name you branch to refer to its purposes (e.g. feature_x or bug_y). Everything you do in git is linked to your account, so no need to indicate this in the branch name. Once merged, the branch can be deleted.

**Comment by malachig** [comment]
+1 this looks good now.  Go ahead and merge as long as @gatoravi is happy.

**Comment by malachig** [comment]
Spoke too soon.  This tool should by default ignore evidence statements that have a status of rejected...

**Comment by gatoravi** [comment]
I think we could remove the file 'Drug_check_usage.txt' file and add the details listed there in the README instead..

**Comment by LeiChen0218** [comment]
Oh I took it for granted that those are all accepted evidences.
It should be fixed now (as well as the merging of two Usage files). You guys can go back and check again.
Poke me if you find anything else to fix.

Have a nice weekend :)
Lei

Sent from my iPhone

On Feb 19, 2016, at 14:26, Malachi Griffith <notifications@github.com<mailto:notifications@github.com>> wrote:

Spoke too soon. This tool should by default ignore evidence statements that have a status of rejected...

##

**Comment by malachig** [comment]
I think this is fine.  Lets go ahead and merge this now.

**Comment by malachig** [comment]
Actually I am getting an error when I try this now:
`civic-api-client evidence-items-list --max-gene-count 10 --evi-type=Predictive --drug`

```
Max number of genes to query is  10
Traceback (most recent call last):
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/bin/civic-api-client", line 11, in <module>
    sys.exit(main())
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/command_line.py", line 24, in main
    return eil1.main()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 216, in main
    self.create_invalid_eis_list()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 207, in create_invalid_eis_list
    self.check_drug_for_pre(variant_id, variant_detail,evidence_items)
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 136, in check_drug_for_pre
    if evidence_items['status'] == 'rejected':
TypeError: list indices must be integers, not str
```

**Comment by LeiChen0218** [comment]
There shouldn't have been an "=" followed by "ebi-type" actually...

Lei

Sent from my iPhone

On Feb 22, 2016, at 10:18, Malachi Griffith <notifications@github.com<mailto:notifications@github.com>> wrote:

Actually I am getting an error when I try this now:
civic-api-client evidence-items-list --max-gene-count 10 --evi-type=Predictive --drug

Max number of genes to query is  10
Traceback (most recent call last):
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/bin/civic-api-client", line 11, in <module>
    sys.exit(main())
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/command_line.py", line 24, in main
    return eil1.main()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 216, in main
    self.create_invalid_eis_list()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 207, in create_invalid_eis_list
    self.check_drug_for_pre(variant_id, variant_detail,evidence_items)
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 136, in check_drug_for_pre
    if evidence_items['status'] == 'rejected':
TypeError: list indices must be integers, not str

##

**Comment by LeiChen0218** [comment]
Ignore my previous email… “=“ also works for argument parsing.
Identified and fixed the bug. It’s so weird that the wrong script actually worked on my local directory...
Would you mind try one more time? (It looks that some Predictive evidences with missing drug error were fixed)

Lei

On Feb 22, 2016, at 10:18 AM, Malachi Griffith <notifications@github.com<mailto:notifications@github.com>> wrote:

Actually I am getting an error when I try this now:
civic-api-client evidence-items-list --max-gene-count 10 --evi-type=Predictive --drug

Max number of genes to query is  10
Traceback (most recent call last):
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/bin/civic-api-client", line 11, in <module>
    sys.exit(main())
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/command_line.py", line 24, in main
    return eil1.main()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 216, in main
    self.create_invalid_eis_list()
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 207, in create_invalid_eis_list
    self.check_drug_for_pre(variant_id, variant_detail,evidence_items)
  File "/Users/mgriffit/git/civic/civic-api-client/myenv/lib/python2.7/site-packages/civic_api_client/evidence_items_lister.py", line 136, in check_drug_for_pre
    if evidence_items['status'] == 'rejected':
TypeError: list indices must be integers, not str

**Comment by gatoravi** [comment]
Ok, this is working well now! Merging this

**Pull request merged**